pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -508,6 +489,29 @@ func (c *component) createOrPatch(ctx context.Context, sts *appsv1.StatefulSet,
 		if c.values.PriorityClassName != nil {
 			sts.Spec.Template.Spec.PriorityClassName = *c.values.PriorityClassName
 		}
+		if c.values.UseEtcdWrapper {
+			// sections to add only when using etcd wrapper
+			// TODO: @aaronfern add this back to sts.Spec when UseEtcdWrapper becomes GA
+			sts.Spec.Template.Spec.InitContainers = []corev1.Container{
+				{
+					Name:         "change-permissions",
+					Image:        "alpine:3.18.2",
+					Command:      []string{"sh", "-c", "--"},
+					Args:         []string{"chown", "-R", "65532:65532", "/var/etcd/data"},
```

Comment:
@aaronfern , this does not actually work. It throws error:
```
> k logs etcd-test-2 -c change-permissions -f
BusyBox v1.36.1 (2023-06-02 00:42:02 UTC) multi-call binary.

Usage: chown [-RhLHPcvf]... USER[:[GRP]] FILE...

Change the owner and/or group of FILEs to USER and/or GRP

	-h	Affect symlinks instead of symlink targets
	-L	Traverse all symlinks to directories
	-H	Traverse symlinks on command line only
	-P	Don't traverse symlinks (default)
	-R	Recurse
	-c	List changed files
	-v	Verbose
	-f	Hide errors
```

You'll have to revert back to `Args: []string{"chown -R 65532:65532 /var/etcd/data"},`. Apologies for requesting this change without testing it.